.gitignore

```diff
@@ -110,3 +110,7 @@ website/public/
 website/resources/
 website/node_modules/
 website/tech-doc-hugo
+
+# samples bin
+testdata/go/memcached-operator/bin/*
```

Comment:
This can be removed for now. We can address in a follow-up.

Reply:
IHMO we should keep in order to ensure that the bin will not be committed never and ever,